wazo_debug/access.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2017-2021 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import argparse
+import logging
+import random
+import time
+import subprocess
+
+from cliff.command import Command
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+
+ssh_max_retries = 3
```

Comment:
```suggestion
ssh_max_retries = 10
```
3 seems low, no? Except if the timeout between retry is high ( >2 sec), I would increase retries

Reply:
The retry loop is only here to avoid random port collision on the remote server. 3 seems a good value to me. If more retries are needed, we can always restart the command.